lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -158,11 +186,13 @@ def self.get_or_create_app(params)
       def self.run(params)
         values = params.values
         upload_dsym_only = params[:upload_dsym_only]
+        upload_mapping_only = params[:upload_mapping_only]
 
         # if app found or successfully created
         if self.get_or_create_app(params)
-          self.run_release_upload(params) unless upload_dsym_only
-          self.run_dsym_upload(params)
+          self.run_release_upload(params) unless upload_dsym_only unless upload_mapping_only
```

Comment:
I didn't know you can do two `unless` like this :)

Reply:
Actually I think it is better to use `unless A || B` here for clarity